docs/bitcoin-core-usage.md

```diff
@@ -0,0 +1,344 @@
+# Using Bitcoin Core with Hardware Wallets
+
+This approach is fairly manual, requires the command line, and requires a patched version of Bitcoin Core.
+
+Note: For this guide, code lines prefixed with `$` means that the command is typed in the terminal. Lines without `$` are output of the commands.
+
+### Disclaimer
+
+I am not liable for any coins that may be lost through this method. The software mentioned may have bugs. Use at your own risk.
+
+## Software
+
+### Bitcoin Core
+
+This method of using hardware wallets uses Bitcoin Core as the wallet for monitoring the blockchain. It allows a user to use their own full node instead of relying on an SPV wallet or vendor provided software.
+
+However, Bitcoin Core needs a few changes for this work.
+PRs [#14689](https://github.com/bitcoin/bitcoin/pull/14689), [#14565](https://github.com/bitcoin/bitcoin/pull/14565), [#14491](https://github.com/bitcoin/bitcoin/pull/14491), [#14021](https://github.com/bitcoin/bitcoin/pull/14021), and [#14075](https://github.com/bitcoin/bitcoin/pull/14075) are required in order for this to work.
+Since these PRs are still unmerged, I have created a branch with them merged [here](https://github.com/achow101/bitcoin/tree/hww).
+
+### HWI
+
+A set of scripts that I have called Hardware Wallet Interaction Scripts (HWI) is necessary for this. These scripts are a driver for using hardware wallets. The scripts can be found [here](https://github.com/achow101/HWI).
+
+## Setup
+
+Clone the modified Bitcoin Core and build it. Clone HWI.
+
+```
+$ git clone https://github.com/achow101/bitcoin.git -b hww
+$ cd bitcoin
+$ ./autogen.sh
+$ ./configure
+$ make
+$ src/bitcoind -daemon -addresstype=legacy -changetype=legacy
```

Comment:
`legacy` type is inconsistent with the use of `bech32` below, and lame in general :-)

Reply:
Fixed